examples/stable-diffusion/quantization/SD3/measure_config.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
looks good, I would use lowercase letters to be more in style of other folder names

Reply:
Updated these